yapftests/reformatter_test.py

```diff
@@ -1914,6 +1914,30 @@ def testExecAsNonKeyword(self):
     self.assertEqual(expected_formatted_code, reformatter.Reformat(uwlines))
 
 
+class TestingNotInParameters(unittest.TestCase):
+
+    def test_notInParams(self):
+        unformatted_code = textwrap.dedent("""\
+            def sum(a, sprint):
```

Comment:
(Though I don't like your test overriding the builtin sum function :p these tests would work just as well with just the last line.... but I don't think this matters.)